### DISCUSSION TITLE
У новых пользователей не работает тотум

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#438] [ОШБК!] [answered]
Добавляю пользователя, создаю роль, добавляю доступы к таблицам, но пользователь ничего не видит, даже не работает нажатие на собственное имя. Плюс если как с Админа переключиться на пользователя, то обратно не получается переключиться.
[test.webm](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/d7af6d5a-b8e3-40dc-85c1-b134b23cf179)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Да, все умерла система переключающая пользователей — посмотрим.

**Reply by ttm-alexeypolunin** [reply]
Исправили — обновите

**Reply by mihail-cornev** [reply]
Спасибо, работает.
